app/views/homeTab/components/Home.js

```diff
@@ -1,23 +1,20 @@
  'use strict'
  import React from 'react'
-import { connect } from 'react-redux'
-import { Container, Content, Text, Button, Form, Item, Icon, Input, List, ListItem, Body, Label, Title, Header } from 'native-base'
-import GroupListItem from '../../groupTab/components/GroupListItem'
-import { fetchGroups } from '../../../redux/reducers/groups'
+ import { connect } from 'react-redux'
+ import { Container, Content, Text, Button, Form, Item, Icon, Input, List, ListItem, Body, Label, Title, Header } from 'native-base'
+ import GroupListItem from '../../groupTab/components/GroupListItem'
+ import { fetchGroups } from '../../../redux/reducers/groups'
 
```

Comment:
Can we fix the indentation in this file? Not sure what broke it but everything has 1 extra space now.